learning_assistant/utils.py

```diff
@@ -0,0 +1,43 @@
+"""
+Utils file for learning-assistant
+"""
+import logging
+import requests
+from requests.exceptions import ConnectTimeout, ConnectionError
+
+from django.conf import settings
+from rest_framework import status as http_status
+
+log = logging.getLogger(__name__)
+
+
+def get_chat_response(message_list):
+    """
+    Pass message list to chat endpoint, as defined by the CHAT_COMPLETION_API setting
+    """
+
+    completion_endpoint = getattr(settings, 'CHAT_COMPLETION_API', None)
+    if completion_endpoint:
+        headers = {'Content-Type': 'application/json'}
+        connect_timeout = getattr(settings, 'CHAT_COMPLETION_API_CONNECT_TIMEOUT', 1)
+        read_timeout = getattr(settings, 'CHAT_COMPLETION_API_READ_TIMEOUT', 10)
+
+        try:
+            response = requests.post(
```

Comment:
I'm not sure if this connection point should be in its own client class, but put it in a utils file for now.